extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -49,6 +49,7 @@ tags: {  # <amp-video>
   }
   attrs: { name: "controlslist" }
   attrs: { name: "crossorigin" }
+  attrs: { name: "disableremoteplayback" }
```

Comment:
Suggestion: if `disableremoteplayback` isn't supposed to have any value to the attribute, may want to make it like `loop`:

```
attrs; {
  name: "disableremoteplayback"
  value: ""
}
```

Reply:
good call, done.